lib/routes/whu/rsgis.ts

```diff
@@ -0,0 +1,283 @@
+            hzjl: {
+                name: '合作交流',
+                path: 'hzjl',
+            },
+            mtjj: {
+                name: '媒体聚焦',
+                path: 'mtjj',
+            },
+            xgyw: {
+                name: '学工要闻',
+                path: 'xgyw',
+            },
+        },
+    },
+    kxyj: {
+        name: '科学研究',
+        path: 'kxyj',
+        sub: {
+            xsbg: {
+                name: '学术报告',
+                path: 'xsbg',
+            },
+            xsjl: {
+                name: '学术交流',
+                path: 'xsjl',
+            },
+            kycg: {
+                name: '学术成果',
+                path: 'kycg',
+            },
+            sbxx: {
+                name: '申报信息',
+                path: 'sbxx',
+            },
+        },
+    },
+    tzgg: {
+        name: '通知公告',
+        path: 'tzgg1',
+        sub: {
+            xytz: {
+                name: '学院通知',
+                path: 'xytz',
+            },
+            jxdt: {
+                name: '教学动态',
+                path: 'jxdt',
+            },
+            xsdt: {
+                name: '学术动态',
+                path: 'xsdt',
+            },
+            rcyj: {
+                name: '人才引进',
+                path: 'rcyj',
+            },
+        },
+    },
+};
+
+/**
+ * Check whether the link is external.
+ *
+ * @param link Post link
+ * @returns Whether or not weixin post
+ */
+function checkExternal(link: string): boolean {
+    const matchWeixin = link.match(/^((http:\/\/)|(https:\/\/))?([\dA-Za-z]([\dA-Za-z-]{0,61}[\dA-Za-z])?\.)+[A-Za-z]{2,6}(\/)/);
+    return !!matchWeixin?.length;
+}
+
+/**
+ * Get information from a list of paired link and date.
+ *
+ * @param element
+ * @returns A list of RSS meta node.
+ */
+function parseListLinkDateItem(element: Cheerio<AnyNode>, currentUrl: string) {
+    const linkElement = element.find('a').first();
+    const title = linkElement.text();
+    const href = linkElement.attr('href');
+    if (href === undefined) {
+        throw new Error('Cannot get link');
+    }
+    const external = checkExternal(href);
+    const link = external ? href : new URL(href, currentUrl).href;
+    const pubDate = element.find('div.date1').first().text();
+    return {
+        title,
+        link,
+        pubDate: timezone(parseDate(pubDate, 'YYYY-MM-DD'), +8),
+        description: title,
+        external,
+    };
+}
+
+async function getDetail(item: Post): Promise<DataItem> {
+    if (item.link) {
+        if (item.external) {
+            item.description = `<a href="${item.link}">阅读原文</a>`;
+        } else {
+            const response = await ofetch(item.link);
+            const $ = load(response);
+            const title = $('div.content div.content_title h1').first().text();
+            const content = $('div.content div.v_news_content').first().html();
+            item.title = title;
+            item.description = content || '';
+        }
+    }
+    return item;
+}
```

Comment:
Please [cache](https://docs.rsshub.app/joinus/advanced/use-cache) the returned object.